src/CMakeLists.txt

```diff
@@ -188,6 +188,11 @@ endif()
 
 find_package(spdlog REQUIRED)
 message(STATUS "spdlog: ${spdlog_VERSION}")
+if(${SPDLOG_FMT_EXTERNAL})
+  message(WARNING "spdlog: SPDLOG_FMT_EXTERNAL=${SPDLOG_FMT_EXTERNAL}")
+else()
+  message(STATUS "spdlog: SPDLOG_FMT_EXTERNAL=${SPDLOG_FMT_EXTERNAL}")
```

Comment:
why are we printing the value when the if failed and it is unset?  This won't be very informative.

Reply:
I would just drop the else clause

Reply:
> I would just drop the else clause

done.

Reply:
> why are we printing the value when the if failed and it is unset? This won't be very informative.
* Previous ```find_package(spdlog REQUIRED)``` would always set the ```SPDLOG_FMT_EXTERNAL``` otherwise would stop/error since it is enforced with ```REQUIRED``` flag.
* But let's remove the ```else``` flag anyway.